functions/firebase/rtdb/app.rb

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_firebase_rtdb]
+require "functions_framework"
+
+# Triggered by a change to a Firebase RTDB document.
+FunctionsFramework.cloud_event "hello_rtdb" do |event|
+  # The event parameter is a CloudEvents::Event::V1 object.
```

Comment:
I'm concerned that users may interpret this as being a format _specific_ to Firebase, as opposed to something shared across multiple triggers.

Maybe re-word to `Non-HTTP Ruby functions receive a CloudEvents::Event::V1 object.`?

Reply:
Updated